### PR TITLE
Add support links to desktop web dex landing/home page

### PR DESCRIPTION
**Pull request by eddieharrison**
## Screenshots
screen shot 2019-02-26 at 16 00 35

### CODE REVIEW

File: src/components/login/Landing.vue [hunk 1]
```diff
@@ -64,6 +65,12 @@ export default {
       }
     }
   }
+
+  .support-links {
+    margin-top: 30px;
```
Comment (apheliontoken) [inline src/components/login/Landing.vue]: You can use:
`margin-top: toRem(30px);`

File: src/components/login/Landing.vue [hunk 2]
```diff
@@ -64,6 +65,12 @@ export default {
...
+    text-align: center;
+    line-height: 1.6rem;
```
Comment (apheliontoken) [inline src/components/login/Landing.vue]: `1.6rem` here is equivalent to: `line-height: toRem(23px);`
For consistency, we are using `toRem()` for the measurements.

File: src/components/login/Landing.vue (hunk 1 above)
Comment (eddieharrison) [reply src/components/login/Landing.vue]: Fixed

File: src/components/login/Landing.vue (hunk 2 above)
Comment (eddieharrison) [reply src/components/login/Landing.vue]: Fixed

**Pull request merged**